src/lib/composite-layer.js

```diff
@@ -25,6 +25,10 @@ export default class CompositeLayer extends Layer {
     super(props);
   }
 
+  get isComposite() {
```

Comment:
It needs to be a getter because layer is sealed? Maybe just keep it as a function?